content/api/commands/getallcookies.md

```diff
@@ -0,0 +1,105 @@
+---
+title: getAllCookies
+---
+
+Get all browser cookies.
```

Comment:
the local storage/ session storage descriptions said `with which the test has interacted.` should we here as well?

Reply:
It's not really the same with cookies. The test doesn't have to interact with a domain to have cookies set on it. With local/session storage, we have to manually track what html the test interacts with to be able to get/clear it. For cookies, we don't since the automation takes care of that.